documentation/app/templates/components/service-list-item.hbs

```diff
@@ -0,0 +1,336 @@
+                }
+              }
+            */
+
+            <Cut::ListItem::Service @service={{this.failingTerminatingGateway}} />
+          "}}}
+          />
+        </PreviewBlock>
+
+        <PreviewBlock as |P|>
+          <P.Preview>
+            <Cut::ListItem::Service
+              @service={{this.mtlsSamenessGroupService}}
+            />
+          </P.Preview>
+          <P.Code
+            @code={{{"
+            /*
+              Example service object
+
+              {
+                name: 'Service: imported, with sameness group, permissive mTLS',
+                metadata: {
+                  healthCheck: {
+                    instance: {
+                      success: 2,
+                      warning: 1,
+                      critical: 0,
+                    },
+                  },
+                  instanceCount: 8,
+                  isImported: true,
+                  samenessGroup: 'group-1',
+                  isPermissiveMTls: true,
+                },
+            */
+            <Cut::ListItem::Service @service={{this.mtlsSamenessGroupService}} />
+          "}}}
+          />
+        </PreviewBlock>
+
+      </div>
+      <InPageNav as |I|>
+        <I.Link @section="#about">About</I.Link>
+        <I.Link @section="#examples">Examples</I.Link>
+      </InPageNav>
+    </div>
+
+  </T.Panel>
+  <T.Panel>
+
+    <div class="doc-Page">
+      <div class="doc-Page-content">
+        <h2 id="component-api">Component API</h2>
+        <p>The Service List Item is composed with the List Item and the List
+          Item Template. It accepts all the arguments that the List Item does.
+          Visit the
+          <Hds::Link::Inline @route="components.list-item">List Item
+            documentation</Hds::Link::Inline>
+          to get full details on it's API.</p>
+        <p>Aside from that, the Service List Item accepts a Service object that
+          is composed of a
+          <CodeInline @code="name" />
+          and a
+          <CodeInline @code="metadata" />
+          object. We'll dive into the details below.</p>
+
+        <CodeBlock
+          @code={{{"
+          interface CutService {
+            name: string | undefined;
+            metadata: {
+              healthCheck: {
+                instance?: HealthCheck;
+              };
+              kind?: SERVICE_KIND;
+              instanceCount?: number;
+              isImported?: boolean;
+              isPermissiveMTls?: boolean;
+              connectedWithGateway?: boolean;
+              connectedWithProxy?: boolean;
+              samenessGroup?: string;
+              externalSource?: ExternalSource;
+              tags: string[];
+              upstreamCount?: number;
+              linkedServiceCount?: number;
+            };
+          }
+          "}}}
+        />
+
+        <ComponentApi @name="name" @isRequired={{true}} @type="string" />
+        <h3 id="metadata">Metadata</h3>
```

Comment:
Looking at the examples in the About tab, is the component arg `@service` with all these nested within that?

Reply:
Yes absolute brain fart. I did talk about it above but it should have a heading of it's own.

Do you think it would make sense to update the argument names to represent that as well? 

So like `service.metadata.kind` instead of `kind`. @natmegs

Reply:
oooh i like that idea!!